panels/user-accounts/data/account-dialog.ui

```diff
@@ -155,6 +155,7 @@
                 <child>
                   <object class="GtkEntry" id="local_name">
                     <property name="visible">True</property>
+                    <property name="max-length">80</property>
```

Comment:
I think these would be better changed as a separate patch. When you rebase this module in a few weeks you'll have to remember to apply this change as a separate patch anyway, no?

Reply:
Updated.